workflows/tree_inference.smk

```diff
@@ -151,4 +151,39 @@ rule mcmc:
         --out_dir {DATADIR}/{wildcards.experiment}/mcmc \
         --data_fp {input.mutation_data} \
         --init_tree_fp {input.init_tree} 
-        """
\ No newline at end of file
+        """
+
+
+# below rule input will trigger gen_cell_simulation rule, which will trigger tree generation rule
+rule run_huntress:
+    """Run HUNTRESS on the true tree.
+
+    - Cell Simulation data requires
+        - no missing entries
+        - no homozygous mutations
+    """
+    input:
+        mutation_data="{DATADIR}/{experiment}/mutations/{mutation_data_id}.json",
+    output:
+        huntrees_tree="{DATADIR}/{experiment}/huntress/HUN-{mutation_data_id}.json"
+    threads: 4 # as many threads as defined in make_huntress.py
```

Comment:
note number of thread hardcoded to 4 for running huntress - is this the right way to do this. ?